git-hub

```diff
@@ -209,35 +294,44 @@ def find_match(open_or_closed, list_of_dictionaries, keyword, user, comment, num
 
     Parameters
     ----------
-    open_or_closed : boolean
+    Open_or_closed : boolean
         Whether looking in closed PR or open PR
-    dictionary : dictionary
+    List_of_dictionaries : dictionary
         pull-requests.toml information mapping PR number to info
-    keyword : string
+    Keyword : string
         Searches if any part of user, branch, comment, or number match
-    user : string
+    User : string
         Search by PR username
-    comment : string
+    Comment : string
         Search by PR comment
-    number : string
+    Number : string
         Search by PR number
-    branch : string
+    Branch : string
         Search by PR branch
-    appeared_before : boolean
+
+    Returns
+    ----------
+    Appeared_before : boolean
         Whether the search match is the first match
+    All_prs : list
+        All the prs that match the criteria
     """
     appeared_before = False
     all_prs = {}
+    GREEN = '\x1b[0;32;40m'
+    RED = '\x1b[0;31;40m'
+    CLOSE = '\x1b[0m'
     for dictionary in list_of_dictionaries:        # iterates through and checks if open pull requsts have keyword
         for sub_keys in list(dictionary.keys()):
             sub_dictionary = dictionary[sub_keys]
             if keyword:         # checks keyword
-                if keyword.upper() in sub_dictionary.keys().upper() or keyword in sub_dictionary.values().upper():
+                concate = " ".join(sub_dictionary.values())
+                if keyword.upper() in concate.upper():
                     appeared_before = True
                     if(open_or_closed):
-                        all_prs[sub_dictionary['modified']] = '{0} \x1b[0;32;40m {1} \x1b[0m {2}/{3} {4}'.format(sub_keys, "O", sub_dictionary['user'], sub_dictionary['branch'], sub_dictionary['comment'])
```

Comment:
Instead of hard-coding ansi sequences, define them in an external file and give them reasonable names.

Another option may be to use a terminal package, such as `blessings`.